src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -360,7 +460,7 @@ EncodedResults StaticLLMPipeline::generate(
 
     auto padded_input_ids = m_prefill_request.get_tensor("input_ids");
     const size_t offset = padded_input_ids.get_size() - input_ids.get_size();
-    copy_with_offset(input_ids, offset, padded_input_ids);
+    copy_with_offset(input_ids, static_cast<int32_t>(offset), padded_input_ids);
```

Comment:
I don't mind if `offset` is `uint32_t` in `copy_with_offset` function. Will it help to get rid of `static_cast`(s)?

Reply:
Fixed, thanks!